controllers/sriovoperatorconfig_controller.go

```diff
@@ -188,6 +188,7 @@ func (r *SriovOperatorConfigReconciler) syncConfigDaemonSet(ctx context.Context,
 		data.Data["UsedSystemdMode"] = false
 	}
 	data.Data["ParallelNicConfig"] = r.FeatureGate.IsEnabled(consts.ParallelNicConfigFeatureGate)
+	data.Data["ConfigDaemonEnvVars"] = dc.Spec.ConfigDaemonEnvVars
```

Comment:
can you add test for this one ?

Reply:
added, see https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/733/commits/76a8f95581b0449074941997f371ac0903139837